app/views/formatted-array.html

```diff
@@ -0,0 +1,13 @@
+<div class="formatted-array">
```

Comment:
maybe we can add a suffix to know which files are directive templates and which are views ?